doc/reference/01_overview.adoc

```diff
@@ -0,0 +1,80 @@
+
+== Overview
+
+Sirius Web provides a modeling workbench to build model based applications.
+There are two kinds of users which will interact with Sirius Web, each one working with a specific aspect of the application.
+Firstly, specifiers will use Sirius Web to define all the concepts of their business and the graphical tooling to manipulate them.
+Secondly, end users which will use those concepts to capture some business knowledge and manipulate those pieces of data thanks to some dedicated tooling.
+
+Sirius Web can thus be described with four major parts.
+The specifiers work at a "meta" level where they describe the business concepts in a domain and the descriptions of the representations in a view.
+The end users will manipulate instances of those concepts thanks to the representations.
+
+image:images/topics.png[]
+
+
+The semantic data manipulated by the end users are thus described by the domains while the graphical representations are described by the views.
+Since the representations are tailor made for the business of the users, they are configured by the concepts defined in the domains.
+End users can thus manipulate their data with the representations.
+
+image:images/relations.png[]
+
+Sirius Web is the descendant of the Sirius "desktop" project.
+Both projects try to fulfill similar requirements and share the same philosphy.
+Sirius Web provides a compatibility layer for the specifiers who have used Sirius desktop in the past.
+
+
+=== Domain
+
+In a domain, specifiers will define all the concepts of their business with their attributes and the various relations between these concepts.
+Specifiers can use the new `Domain` language exclusive to Sirius Web to specify those concepts.
+They can also leverage existing `ecore` files that may have been created for Sirius desktop.
+
+Since most of the time semantic data are manipulated using the Eclipse EMF framework, domains are sometimes referred as metamodels or even `EPackages`.
+
+Using a domain, a specifier could create concepts such as `Vehicule` and `Engine` with their properties `name`, `speed`, `energyType`, `numberOfWheels` etc.
+The specifier should have a deep understanding of the business and its various rules.
+
+
+=== View
+
+In a view, specifiers will create the descriptions of the representations they want to provide to their end users.
+The view can be used to describe various representations such as diagrams or forms to edit the data.
+
+Sirius Web comes with the `View` language to create and parameterize the various representations.
+In Sirius desktop, specifiers would use an `odesign` file to create the descriptions of the representations.
+It is possible to import those `odesign` files in Sirius Web to reuse some tooling created in Sirius desktop.
+
+Using a view, a specifier can create the description of a diagram which will be used by the end users to define various kinds of vehicules.
```

Comment:
Typo (here and in other instances below): `vehicles` (no `u`)

Reply:
I've fixed that and other typos